plugins/admin/pkg/command/registry/add.go

```diff
@@ -124,9 +124,9 @@ kn admin registry add \
 
 			_, err = p.ClientSet.CoreV1().ServiceAccounts("default").Update(desiredSa)
 			if err != nil {
-				return fmt.Errorf("failed to add registry secret in default serviceaccount: %v", err)
+				return fmt.Errorf("Failed to add registry secret in default serviceaccount: %v", err)
```

Comment:
Why is the message capitalized here but lower case before. Let’s be consistent.

Reply:
Thanks Max! Fixed.